documentation/user_docs/docs/manual/Correcting_for_sample_misalignment.rst

```diff
@@ -193,50 +226,89 @@ The keyword options are:
 
-Once you have confirmed that the alignment you have is the correct one, it is possible to fix the alignment to avoid this calculation step.
+This procedure modifies lattice parameters and adds alignment matrix to the pixels data. When ``change_crystal`` is used on ``sqw`` file, pixels themselves are not modified so the alignment procedure is very fast. Pixels will be aligned whenever they are loaded or manipulated (e.g. accessing pixel data, cutting, plotting, etc.). The pixels alignment is combined with other transformations, usually performed during pixels manipulations, so the speed of majority of such operations is not affected. The actual slow-down in operations with aligned file occurs when some advanced algorithms use pixels range (e.g. ``mask_pixels`` based on a range). Pixels range is invalidated when pixels are realigned, so such algorithms have to calculate this range. This may take substantial time. Majority of Horace users do not need to use such algorithms so may work with fast-realigned files without any noticeable hindrance.
 
-This is done through the ``finalize_alignment`` function:
+If you are following :ref:`iterative process<StepA>` above, after validating your alignment revert your alignment at :ref:`Step D <StepD>` applying:
+
+::
+
+   >>change_crystal(win, reverse_transf);
+   or 
+   >>wout = change_crystal(wout, reverse_transf);
+
+
+If you performed multiple alignment and ``change_crystal`` operations on filebacked object without reverting them, you may recover resulting reverse (or direct) transformation from filebacked object's pixels alignment matrix:
+
+::
+
+    >>reverse_transf = crystal_alignment_info(alatt0,angdeg0);
+    >>reverse_transf.rotmat = wout.pix.alignment_matr';
+
+This is possible because resulting alignment (and de-alignment) matrix is the result of multiplication of sequence of rotation operations.
+
+There are no possibility to retrieve lost initial lattice parameters ``alatt0``; ``angdeg0`` from any ``sqw`` object and alignment matrix from memory based aligned ``sqw`` object.
+This is why it is recommended to revert the alignment first each time you want to realign your ``sqw`` object. It is not the critical recommendation, as you can always rebuild your misaligned ``sqw`` object from the initial experimental results.
+
+Once you have confirmed that the alignment you have is the correct one, it is possible to fix the alignment to avoid pixel ranges calculation step mentioned above.
+
+This is done using the ``finalize_alignment`` function:
 
 ::
 
    [wout, rev_corr] = finalize_alignment(win, ['-keep_original'])
 
-.. warning::
+.. Note::
 
-   You must have attached the alignment to the ``sqw`` through the ``change_crystal`` function prior to applying it.
+   You must have attached the alignment to the ``sqw`` through the ``change_crystal`` function prior to applying it, as it will do nothing otherwise.
```

Comment:
put this line after line 325 so the argument descriptions are next to the function call examples